### DISCUSSION TITLE
Demo App using VueJS and Quasar

### DISCUSSION THREAD

**Discussion by alainrc2005** [adonisjs/core#1430] [Show and tell]
I created a demo app using VueJS and Quasar integration. [adonisjs-quasar-red](https://github.com/alainrc2005/adonisjs-quasar-red).
-[Quasar Table component server-side paging](http://alainrc2005.blogspot.com/2020/08/paginacion-del-lado-del-servidor-quasar.html)

It is currently 98% complete

View it out, and give it a star!

**Comment by armgitaar** [comment]
This looks pretty cool! I tried running and looks like migration was missing `mw_visits` table and columns. After getting past that, I had a bit of trouble authenticating. Let me know when you are 100%, I'd like to take a look again!

**Reply by alainrc2005** [reply]
Solved, please update...At the moment I only have two functionalities pending: forgotPassword and changePassword

It is a system for registering points of computer networks of entities

**Reply by alainrc2005** [reply]
The system already has all the basic functionalities implemented, some seeds are missing for data visualization. Any problem that arises, you can contact me to solve it. It is a demonstration implementation for the community
